### PR TITLE
Newsletter-45: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

File: _posts/zh/newsletters/2019-05-07-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  提案发布的时间太晚，无法让我们在本期 Newsletter 中提供详细描述，尽管我们确实修改了本期 Newsletter 中的其他一些文字以反映提案的发布。我们计划在下周提供完整的报道。
+
+## Bech32 发送支持
+
+*第 8 周，共 24 周。在 2019 年 8 月 24 日 segwit 软分叉锁定的第二周年纪念日之前，Optech Newsletter 将包含这个每周部分，提供帮助开发者和组织实现 bech32 发送支持——支付本机 segwit 地址的能力的信息。这[不需要你自己实现 segwit][bech32 series]，但它确实允许你支付的人访问 segwit 的所有好处。*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/zh/08-upgradability.md %}
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2019-05-07-newsletter.md:64]: @bittomhan can you add this zh/08-upgradability.md reference to the zh/bech32 post page as well?

Comment (bittomhan) [reply _posts/zh/newsletters/2019-05-07-newsletter.md:64]: Thank you! I have completed the modifications. In the future, I will perform a rebase on each submission sequentially.

File: _posts/zh/2019-03-19-bech32-sending-support.md [hunk 2]
```diff
@@ -178,7 +178,8 @@ h2:not(:first-of-type) { margin-top: 3em; }
 [newsletter #42]: /zh/newsletters/2019/04/16/#bech32-发送支持
 [newsletter #43]: /zh/newsletters/2019/04/23/#bech32-发送支持
 [newsletter #44]: /zh/newsletters/2019/04/30/#bech32-发送支持
-[newsletter #45]: /en/newsletters/2019/05/07/#bech32-sending-support
+[newsletter #44]: /zh/newsletters/2019/04/30/#bech32-发送支持
```
Comment (bitschmidty) [inline _posts/zh/2019-03-19-bech32-sending-support.md:181]: I think this line is a duplicate

Comment (bittomhan) [reply _posts/zh/2019-03-19-bech32-sending-support.md]: Indeed, that's correct! Thank you for checking. I tried it again locally, and this kind of link duplication cannot actually be detected by make production. This error occurred when I was performing git rebase -master. I apologize for the inconvenience; I'm still not very proficient with git operations at times.

File: _posts/zh/newsletters/2019-05-07-newsletter.md (hunk 1 above)
Comment (bitschmidty) [reply _posts/zh/newsletters/2019-05-07-newsletter.md:64]: @bittomhan this doesnt appear to be in place. Still referencing the en version:
https://github.com/bitcoinops/bitcoinops.github.io/pull/1754/files#diff-d7e4457adb7e4fd45a53f9219da2f3c80db818edfb9e191f0d4e9295af002807L73

**Review by bitschmidty — APPROVED** [review]
ACK

**Pull request merged**